rs/consensus/src/consensus.rs

```diff
@@ -148,11 +148,13 @@ impl ConsensusImpl {
         message_routing: Arc<dyn MessageRouting>,
         state_manager: Arc<dyn StateManager<State = ReplicatedState>>,
         time_source: Arc<dyn TimeSource>,
-        stable_registry_version_age: Duration,
+        registry_poll_delay_duration_ms: u64,
```

Comment:
nit: could we change the type to `Duration`?

Reply:
Let me do this in a follow-up MR because passing this constant from so deep also doesn't make much sense.